package.json

```diff
@@ -10,10 +10,12 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "https://sindresorhus.com"
 	},
+	"contributors": ["Jovica Aleksic <jovica.aleksic@loopmode.de>"],
```

Comment:
I don't use this field. It's not actually used anywhere and it's creates inequality with contributors that doesn't add themselves there. I prefer to keep it simple and just not use it. Hope you understand :)

Reply:
Absolutely no problem! :)